knoten/__init__.py

```diff
@@ -1,20 +1,26 @@
 import ctypes
 from ctypes.util import find_library
-from distutils import sysconfig
+from glob import glob
 import os
 import warnings
 
-from . import csm
-
-from csmapi import csmapi
```

Comment:
I think these lines need to stay for passing tests, that or the imports in the failing tests need to change

Reply:
So, OS X is passing. It's this line on a linux system: https://github.com/DOI-USGS/knoten/pull/124/files#diff-9845ec9ebca5ff5318d96b72dea2063013cf4baf376e5a14c7f19a49051f8bdaR18

Where the globbed `results[-1]` are being grabbed. I bet that result is None.... I'll looks some more tomorrow AM.

Reply:
Resolving: The issue was that the glob was hard coded for OSX/Darwin. It is searching for `*.#.#.#.dylib`. That obviously fails on linux. I swapped the search pattern and put in checks for linux/Darwin OS.

I did not look to see what the windows installation naming looked like for usgscsm. That will need to be added in a PR as well.